### PR TITLE
Print dataset info in tests.

### PR DESCRIPTION
**Pull request by xuhdev**

### CODE REVIEW

**Comment by xuhdev** [comment]
@djalova Looks like we should have merged your license file change first

File: tests/test_pydax.py [hunk 1]
```diff
@@ -45,4 +45,6 @@
     # Sanity check. In case of all tests being skipped because of a minor error such as in formatting.
     assert len(versions) > 0
     for version in versions:
+        # Print dataset info. This also examines relevant portion in license.yaml
+        print(pydax.get_dataset_metadata(name, version, human=True))
```
Comment (edwardleardi) [inline tests/test_pydax.py:49]:
```suggestion
        print(pydax.get_dataset_metadata(name, version, human=True), end='\n\n')
```

**Review by edwardleardi — APPROVED** [review]
small suggestion

**Review by edwardleardi — APPROVED** [review]
LGTM

**Pull request merged**